code/modules/mining/laborcamp/laborstacker.dm

```diff
@@ -127,15 +133,10 @@
 
 
 /obj/machinery/mineral/stacking_machine/laborstacker
-	var/points = 0 //The unclaimed value of ore stacked.  Value for each ore loosely relative to its rarity.
-	var/list/ore_values = list("glass" = 1, "metal" = 2, "reinforced glass" = 4, "gold" = 20, "silver" = 20, "uranium" = 20, "titanium" = 20, "solid plasma" = 20, "plasteel" = 23, "plasma glass" = 23, "diamond" = 25, "bluespace polycrystal" = 30, "plastitanium" = 45, "bananium" = 50)
+	var/points = 0 //The unclaimed value of ore stacked.
 
 /obj/machinery/mineral/stacking_machine/laborstacker/process_sheet(obj/item/stack/sheet/inp)
-	if(istype(inp))

```

Comment:
This proc is only called from here
https://github.com/tgstation/tgstation/blob/82386a86f8f1bd7581f938953911938a33635438/code/modules/mining/machine_stacking.dm#L87-L88
so the istype check is redudant.

Reply:
thanks for checking on that